crates/tauri/src/scope/fs.rs

```diff
@@ -123,7 +125,7 @@ fn push_pattern<P: AsRef<Path>, F: Fn(&str) -> Result<Pattern, glob::PatternErro
 
   if let Some(p) = canonicalized {
     list.insert(f(&p.to_string_lossy())?);
-  } else if cfg!(windows) {
+  } else if cfg!(windows) && !path.to_string_lossy().starts_with("\\\\") {
```

Comment:
My brain is still in vacation mode but couldn't we end up with no allowed paths with this check?

Also i think this whole issue calls for new test cases if you wouldn't mind adding them :)

Reply:
The un-canonicalized path is added to the list above (line 81). A Windows path that starts with `\\` is already a UNC, device, or verbatim path and prefixing it with `\\?\` doesn't make sense.

I'll work on some test cases.

Reply:
Ahh damn, missed that line, sry.